Pod/Classes/URBNScrollSyncCollectionView.m

```diff
@@ -48,6 +48,10 @@ - (void)sharedInit
     NSAssert([self.collectionViewLayout isKindOfClass:[UICollectionViewFlowLayout class]], @"Cannot use URBNScrollSyncCollectionView with non flow layout");
     self.animateScrollSync = NO;
     self.delegate = self;
+
+    if ([[[UIDevice currentDevice] systemVersion] compare:@"9.0" options:NSNumericSearch] == NSOrderedAscending) {
+        [self setUpPassThroughDelegateWithDelegate:self.delegate];
```

Comment:
Currently, this is the ovverride method causing the crash.  it's the same behavior for 7 and 8 as before, under 9 I'm not allowing that override to go through